onnx/backend/test/case/node/topk.py

```diff
@@ -38,3 +38,35 @@ def export_top_k():  # type: () -> None
 
         expect(node, inputs=[X, K], outputs=[values_ref, indices_ref],
                name='test_top_k')
+
+    @staticmethod
+    def export_top_k_smallest():  # type: () -> None
+        mode = 0
+        sorted = 1
+
+        node = onnx.helper.make_node(
```

Comment:
This place needs an reference implementation and the output should be computed using the reference implementation.